### DISCUSSION TITLE
Modifica tipologia FC

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#490] [Richieste di aiuto] [answered]
Buongiorno, volevo sapere se fosse possibile aggiungere delle tipologie al Fuori Classe, oltre quelle già previste.
Volevo inoltre sapere se può essere rimosso il limite di inserire il fuori classe dal lato coordinatore solo per date future: capitano a volte delle dimenticanze e, in questo modo, il FC va inserito dal lato docente alunno per alunno..

**Comment by trinko** [comment]
Per quanto riguarda i tipi di fuori classe, se sono di uso generale li posso aggiungere al codice. Quali sarebbero?

Per l'inserimento FC lato coordinatore: si, anche nella mia esperienza il limite non è stato di alcuna utilità e verrà rimosso. Non so quando però, perché richiede modifiche e controlli aggiuntivi (inserire un FC in una data passata/presente significa anche rimuovere eventuale assenza).

**Reply by Lucaccc** [reply]
Le tipologie di fuori classe che vorrei inserire non sono propriamente un fuori classe, ma credo che abbia la stessa valenza.
Ad esempio, quando c’è un uscita didattica o un Viaggio d’Istruzione, questa andrebbe segnalata sul registro elettronico, altrimenti gli studenti risulterebbero presenti in aula quando in realtà non ci sono.
Quindi, le tipologie di cui necessito al momento sono: Uscita Didattica e Viaggio d’Istruzione, che andrebbero un po’ a completare la voce “Attività esterna” specificandone meglio la tipologia.
Se dovessi avere altre idee di fuori classe la aggiorno.

**Comment by trinko** [comment, marked as answer]
Ho visto che su altri registri vengono inserite anche queste tipologie come Fuori Classe, anche se non lo sono. Penso però di aggiungere la gestione anche di queste tipologie per l'anno prossimo. 
La segno come richiesta.
